src/sandstorm/spk.c++

```diff
@@ -889,11 +889,99 @@ private:
         "      )\n"
         "    ],\n"
         "\n"
-        "    continueCommand = .myCommand\n"
+        "    continueCommand = .myCommand,\n"
         "    # This is the command called to start your app back up after it has been\n"
         "    # shut down for inactivity. Here we're using the same command as for\n"
         "    # starting a new instance, but you could use different commands for each\n"
         "    # case.\n"
+        "\n"
+        "    metadata = (\n"
+        "      # Data which is not needed specifically to execute the app, but is useful\n"
+        "      # for purposes like marketing and display.  These fields are documented at\n"
+        "      # https://docs.sandstorm.io/en/latest/developing/publishing-apps/#add-required-metadata\n"
+        "      # and in the source code in the Metadata section of\n"
+        "      # https://github.com/sandstorm-io/sandstorm/blob/master/src/sandstorm/package.capnp\n"
+        "      icons = (\n"
+        "        # Various icons to represent the app in various contexts.\n"
+        "        appGrid = (svg = embed \"path/to/128x128.svg\"),\n"
```

Comment:
Suggest: Comment out the icons to avoid it not compiling; use `appGrid` in the appGrid*svg filename to indicate the purpose of the icon